Factory/DiactorosFactory.php

```diff
@@ -65,6 +65,7 @@ public function createRequest(Request $symfonyRequest)
             ->withCookieParams($symfonyRequest->cookies->all())
             ->withQueryParams($symfonyRequest->query->all())
             ->withParsedBody($symfonyRequest->request->all())
+            ->withRequestTarget($symfonyRequest->getRequestUri())
```

Comment:
Note that a HTTP request begins with a request target (e.g `GET /request-target?query=string`) **not** a URL.

In the same way, the [PSR-7 Request interface](https://github.com/php-fig/http-message/blob/master/src/RequestInterface.php) does not mention URLs or constructors. It only knows about request targets. Therefore, I think we should be explicit about setting the request target correctly using [`withRequestTarget`](https://github.com/php-fig/http-message/blob/master/src/RequestInterface.php#L61) rather than relying on a particular implementation's constructor